guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li> After these <s>operations</s> <strong>operations,</strong> we get an array of points of the shape (P, K, 3+E)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
''corrected accordingly'', Thanks.